packages/core/src/App/Components/Animations/fade-wrapper.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes      from 'prop-types';
+import React          from 'react';
+import posed,
+{ PoseGroup }         from 'react-pose';
```

Comment:
extra space

Reply:
@mustofa-binary I copied this file from trader as per Mahdi, so will keep it identical for now.